sw/device/tests/sram_ctrl_execution_test_main.c

```diff
@@ -127,7 +129,7 @@ static void sram_execution_test(void) {
   ram_exec_function_t func = (ram_exec_function_t)execution_test_instructions;
 
   uintptr_t func_address = (uintptr_t)func;
-  CHECK(func_address > kRamStartBytes && func_address <= kRamEndBytes,
+  CHECK(func_address = > kRamStartAddr && func_address <= kRamEndAddr,
```

Comment:
I think we will need to do this in several places. Suggest adding a reusable API:
```c
typedef struct add_range {
  uint32_t start;
  uint32_t end;
} addr_range_t;

/** 
  * Indicates whether the provided chunk is within an address range.  
  */
bool is_in_range(addr_range_t chunk, addr_range_t range) {
  return (chunk.start <= chunk.end && 
          chunk.start >= range.start && chunk.end <= range.end);
}
```

Reply:
Can add this as a testutil, but I think this probably should be a follow-up change.